server/src/main/java/org/elasticsearch/cluster/metadata/MetaData.java

```diff
@@ -949,15 +949,20 @@ public static Builder builder(MetaData metaData) {
         private Settings transientSettings = Settings.Builder.EMPTY_SETTINGS;
         private Settings persistentSettings = Settings.Builder.EMPTY_SETTINGS;
 
-        private final ImmutableOpenMap.Builder<String, IndexMetaData> indices;
+        private final ImmutableOpenMap.Builder<String, IndexMetaData> indicesBuilder;
```

Comment:
Could you undo this renaming? It adds a bunch of noise and I think the name was more consistent before.